PendingReleaseNotes

```diff
@@ -100,3 +96,7 @@
 * Now when noscrub and/or nodeep-scrub flags are set globally or per pool,
   scheduled scrubs of the type disabled will be aborted. All user initiated
   scrubs are NOT interrupted.
+
+* Alpine build related script, document and test are removed, as the most
+  updated APKBUILD script of Ceph is already included by Alpine Linux's
+  aports repository.
```

Comment:
```suggestion
* Alpine build related script, documentation and test have been removed since the most
  updated APKBUILD script of Ceph is already included by Alpine Linux's
  aports repository.
```

Reply:
@neha-ojha thanks. i incorporated the suggested change in my latest revision and repushed.